hedera-node/src/main/java/com/hedera/services/state/exports/SignedStateBalancesExporter.java

```diff
@@ -109,25 +105,36 @@ public SignedStateBalancesExporter(
 		this.signer = signer;
 		this.expectedFloat = properties.getLongProperty("ledger.totalTinyBarFloat");
 		this.dynamicProperties = dynamicProperties;
+		exportPeriod = dynamicProperties.balancesExportPeriodSecs();
+	}
+
+	private Instant getFirstExportTime(Instant now, final int exportPeriodInSecs) {
```

Comment:
Should this function name be .. `getExportTime` since we are using to get nextExportTime as well ?

Reply:
No. The name is to emphasis this function is only invoked at the beginning and only once for the whole execution. 
The `getExportTime` function is not really needed, just for hiding private data member in unit test. It may serve some query purpose in future.